.github/workflows/lint-test.yaml

```diff
@@ -7,24 +7,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
         with:
           fetch-depth: 0
 
       - name: Set up Helm
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 # v3.5
         with:
-          version: v3.6.3
+          version: 'latest'
```

Comment:
version is default to latest (stable) as per https://github.com/Azure/setup-helm#example
Do we still need to specify this?

Reply:
```suggestion
          version: latest
```
It's best to be explicit.